### PR TITLE
Add cron job script for data updates

### PR DESCRIPTION
**Pull request by katekimani**
**What PR does**
- Adds a script to be executed by the crontab.
- Adds instructions to the guide on how to set the crontab on the EC2 instance.

### CODE REVIEW

**Review by joebrew — CHANGES_REQUESTED** [review]
@katekimani Looking good so far. A bit confused about the Rscript section (ie, how to reference correctly in the shebang, etc.). Can you ssh into the instance at at ubuntu@bohemia.systems and see if you can get this set up?

File: scripts/run_odk_get_data_cron.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+ROOT_FOLDER_PATH=/bohemia # TODO Update this to real path
+# Runs odk_get_data for instance id's not already retrieved.
+echo "Started the data retrieval"
+${ROOT_FOLDER_PATH}/scripts/update_database.R
```
Comment (joebrew) [inline scripts/run_odk_get_data_cron.sh:5]: I think this needs some modification. Not exactly sure how (ie, how to run the script via command line: https://stackoverflow.com/questions/18306362/run-r-script-from-command-line). If you can figure it out, go for it; if not, happy to take over. Just let me know

Comment (katekimani) [reply scripts/run_odk_get_data_cron.sh:5]: I used the option of adding the `#!/usr/bin/RScript` to the R code to make this work.

File: scripts/run_odk_get_data_cron.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+ROOT_FOLDER_PATH=/bohemia # TODO Update this to real path
```
[the remark this answers is not in the document]
Comment (katekimani) [reply scripts/run_odk_get_data_cron.sh]: @joebrew Confirmed the cron works correctly on the server.
```
Oct  6 10:15:01 ip-10-0-0-159 CRON[19971]: (root) CMD (sh /home/ubuntu/Documents/bohemia/scripts/run_odk_get_data_cron.sh)
Oct  6 10:15:01 ip-10-0-0-159 CRON[19972]: (root) CMD (command -v debian-sa1 > /dev/null && debian-sa1 1 1)
Oct  6 10:15:01 ip-10-0-0-159 CRON[19973]: (ubuntu) CMD (sh /home/ubuntu/Documents/bohemia/scripts/run_odk_get_data_cron.sh)
Oct  6 10:15:01 ip-10-0-0-159 CRON[19969]: (CRON) info (No MTA installed, discarding output)
Oct  6 10:15:01 ip-10-0-0-159 CRON[19970]: (CRON) info (No MTA installed, discarding output)
```

**Review by joebrew — APPROVED** [review]

**Pull request merged**